server.js

```diff
@@ -24,9 +17,116 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+const Animal = mongoose.model("Animal", {
+  name: String,
+  scientificName: String,
+  doesItFart: Boolean,
+  doesItMaybeFart: Boolean,
+  descriptionExists: Boolean,
+  descriptionNotes: String,
+  paperLink: String,
+  verifiedBy: String,
+});
+
+// if (process.env.RESET_DB) {
+// const seedDatabase = async () => {
+//   await Animal.deleteMany();
+//   doItBeFartin.forEach((animal) => {
+//     new Animal(animal).save();
+//   });
+// };
+// seedDatabase();
+// }
+
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1) {
+    next();
+  } else {
+    res.status(503).json({ error: "Service unavailable" });
+  }
+});
+
+// Route explanation
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send([
+    { "/": "Home" },
+    { "/animals": "All the animals" },
+    { "/animal/:animal": "Find one animal" },
+    { "/description/:boolean": "True or false if there a fart description" },
+    { "/doitbefartin/:boolean": "True or false if it be fartin" },
+    {
+      "/doitmaybebefartin/:boolean": "True or false if it MAYBE be fartin",
+    },
+  ]);
+});
+
+// Route for all the animals
+app.get("/animals", async (req, res) => {
+  let animals = await Animal.find();
+  res.status(200).json({
+    data: animals,
+    success: true,
+  });
+});
+
+// Route for searching for a single animal by name
+app.get("/animal/:animal", async (req, res) => {
+  let animals = await Animal.find();
+
+  animals = animals.filter((a) =>
+    a.name.toLowerCase().includes(req.params.animal.toLocaleLowerCase())
```

Comment:
Nice piece of code, adding both `toLowerCase` and `includes` - making it even more usable. Since this part shall return a single unique animal I think you could have used `findOne` instead of `find`, just to make it a little bit more semantic :-)